### DISCUSSION TITLE
TEMPLATE - Standards IEP Submission 9999

### DISCUSSION THREAD

**Discussion by MatYarger** [DigitizingForImpact/IEPs#5] [Standards IEP]
---
Type: S
Sub-Type: <Methodology | Interface | Verification | Infrastructure | API>
Status: <Draft | Active | Accepted | Provisional | Deferred | Rejected | Withdrawn | Final | Superseded>
IEP: <IEP Number> 
Title: <IEP Title>
Author: <list of authors` real names and optionally, email addrs>
Sponsor: <Name of sponsor organization(s)>
Discussion-to: <URL of current discussion thread>
Created: <date created on, in dd-mmm-yyyy format>
---

## About 
Standards track IEPs are meant to outline a new feature or implementation aligned with the Digitizing for Impact ecosystem.  
  
## Industry Focus
Not to be confused with IEP Type or Sub-Type. This should identify if there is a particular industry that this IEP is focused on. For example, if you are submitting a clean cook stove DMRV methodology for review, then that should be identified here. If it's a mitigation or adaptation methodology, includes MRV of co-benfits, or more, then that should also be identified here. Provide as much detail as possible as it will only support your approval process.

## Abstract
A brief description of the IEP that is being proposed. This should include a breakdown of the project if your methodology has been implemented into a real world project, or the number of projects its been implemented into if there are more than one. If it's an interface enhancement then ensure you outline prior state and the results of the implemented enhancement to collection or monitoring processes in place. Of particular interest are submission currently on digitizing verification and enhancing the understanding of digitizing verification, or verifying digitization. If it is aligned with an infrastructure or API enhancement to increase interoperability, then ensure prior state is identified and how your IEP improves on prior state to solve an identified problem. 

## Rationale 
A breakdown of the rationale as to why this proposal is being suggested, including any metrics, papers, examples or discussion material leading to the proposal creation. 

## Specification 
Technical details of the proposed changes. For example an API change should have the appropriate breakdown of required variables, types and return values. 

## Other Concerns
References to related and previous IEP's and discussions, or will be affected by this change. Security concerns or implications should be expanded on in this section as well. 
  
## Footnotes
A collection of footnotes cited in the IEP, and a place to list non-inline hyperlink targets.
Here is a simple footnote[^1].
A footnote can also have multiple lines[^2].  
You can also use words, to fit your writing style more closely[^note].

[^1]: My reference.
[^2]: Every new line should be prefixed with 2 spaces.  
  This allows you to have a footnote with multiple lines.
[^note]:
    Named footnotes will still render with numbers instead of the text but allow easier identification and linking.  
    This footnote also has been made with a different syntax using 4 spaces for new lines.
  
## Copyright / License
Each new IEP must be placed under a dual license using an open source public domain [Open Source Approved License](https://opensource.org/licenses) and [CC0-1.0-Universal](https://choosealicense.com/licenses/cc0-1.0/)

**Comment by MatYarger** [comment]
Adjust the template above to reflect Gold Standard Methodology Template and Requirements as outlined in this document. 
[Methodology Template 042021.docx](https://github.com/DigitizingForImpact/IEPs/files/10221370/Methodology.Template.042021.docx)

**Reply by Javier-Castro-Viteri** [reply]
Document not possible to be open, please upload again
